awx/connection.go

```diff
@@ -533,3 +534,49 @@ func (c *Connection) indent(data []byte) []byte {
 	}
 	return buffer.Bytes()
 }
+
+var passwordFilterRegex = regexp.MustCompile("(?i)(password|token|authorization|key)")
```

Comment:
Do we need the extra match group?
Please use `"(?i:password|token|authorization|key)"`

(I think according to the [doc](https://github.com/google/re2/wiki/Syntax) the current syntax should only effect the current group?)